index.html

```diff
@@ -0,0 +1,107 @@
+      </li>
+      <li class="navbarListItem"><a class="navbarLink" href="./about.html">About</a></li>
+      <!--I believe this anchors should take the user to other pages, just like the about anchor above this comment does,
+      but for the purpose of this projects, does pages are not done yet. so the anchors are empty, to be compleated on a 
+      later date-->
+      <li class="navbarListItem"><a class="navbarLink">Program</a></li>
+      <li class="navbarListItem"><a class="navbarLink">Join</a></li>
+      <li class="navbarListItem"><a class="navbarLink">Sponsor</a></li>
+      <li class="navbarListItem"><a class="navbarLink">News</a></li>
+      <li class="navbarListItem lastItem"><a>Spring Festival</a></li>
+    </ul>
+  </nav>
+  <main>
+    <section class="headline">
+      <div class="headlineDiv">
+        <p class="miniTitle">"Hello! Music World"</p>
+        <h1 class="pageTitle">
+          CORDOBA CLASICAL<br/> SPRING FESTIVAL <br/>2023
+        </h1>
+        <p class="titleDescription">
+          Immerse yourself in the melodies of Beethoven, Mozart, 
+          and Handel at the Cordoba Spring Festival.
+          Experience the beauty of classical music in a captivating setting.
+        </p>
+        <h2 class="dateOfConcert">
+          2023.7.7(FRI) ~ 8(SAT)
+        </h2>
+        <p class="places">
+          @ Cordoba Real Theater, Art Center Güemes and more
+        </p>
+      </div>
+    </section>
+    <section class="program" id="sectionLink1">
+      <h3 class="programSectionTitle">
+        Main Program
+      </h3>
+      <div class="musicDiv" id="musicProgram">
+      </div>
+      <div>
+        <button class="joinConcertBtn">
+          Join Cordoba Clasical Spring Festival 2023
+        </button>
+      </div>
+    </section>
+    <section class="featuredMusicians" id="sectionLink2">
+      <h3 class="director">
+        Musicians
+      </h3>
+      <div class="cardsDiv" id="musiciansCardsGroup">
+      </div>
+      <button class="moreBtn" id="moreCardsBtn">
+        MORE
+      </button>
+    </section>
+    <footer class="homeFooter">
+      <article class="sponsors">
+        <div class="sponsorTitleDiv">
+          <p class="sponsorTitle">Partner</p>
+        </div>
+        <ul class="sponsorList">
+          <li class="sponsorListItem"><img src="./assets/footer/mozillaIcon.png" class="sponsorIcons " alt="Mozilla Icon"></li>
+          <li class="sponsorListItem"><img src="./assets/footer/googleIcon.png" class="sponsorIcons iconMargin" alt="Google Icon"></li>
+          <li class="sponsorListItem"><img src="./assets/footer/naverIcon.png" class="sponsorIcons" alt="naver Icon"></li>
+          <li class="sponsorListItem"><img src="./assets/footer/daumkakaoIcon.png" class="sponsorIcons iconMargin" alt="daumkakao Icon"></li>
+          <li class="sponsorListItem"><img src="./assets/footer/airbnbIcon.png" class="sponsorIcons iconMargin iconSize" alt="airbnb Icon"></li>
+        </ul>
+      </article>
```

Comment:
- [ ] Great job implementing the `partner` section :+1: However it would be nice to remove them from the `footer` tag for semantic reasons. You can read more about how to use `footer` tag [here](https://www.w3schools.com/tags/tag_footer.asp)